Realm/RLMObjectBase.mm

```diff
@@ -445,3 +448,31 @@ + (NSArray *)requiredPropertiesForClass:(Class)cls {
 }
 
 @end
+
+@interface RLMObjectBase (Handover) <RLMThreadConfined_Private>
+@end
+
+@implementation RLMObjectBase (Handover)
+
+- (realm::AnyThreadConfined)rlm_handoverData {
+    // FIXME: Once merged, should be easier to get schema ref (https://github.com/realm/realm-cocoa/pull/3941)
```

Comment:
Yup, already working on rebasing over that! It requires rather significant changes to this PR 😝